playbooks/09-compliance.yml

```diff
@@ -0,0 +1,22 @@
+- name: Run compliance scan for controllers
```

Comment:
misleading? This is from the hypervisor, not controller node (unless it's the zuul "controller"?).
It seems to run, specifically, the operators scan - maybe something like `Run operators compliance scans` would be closer to the action? (Same goes for L5 in this same file).